packages/react-scripts/template/src/components/example/ArtifactsViewer.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
-import { colors, DataPreview, Skeleton } from '@fs/zion-ui'
+import { colors, LayoutBand, Skeleton } from '@fs/zion-ui'
 import { css } from '@emotion/core'
-import axios from '@fs/zion-axios'
+import useZionAxios from './UseZionAxios'
```

Comment:
we normally use capitalized files just for components, not for utilities. It's annoying to change case in git, so I'm conflicted. My OCD wants to fix it, but maybe it doesn't matter.

Reply:
If it's not the way we want people coding, it shouldn't be in the example app.

Reply:
That's good to know. I will try and fold it in with other changes.